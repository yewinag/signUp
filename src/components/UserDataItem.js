import React, { Component } from 'react'
import { Text, View } from 'react-native'

const UserDataItem = (props)=> (
            <View>
                <Text>{'   '} {props.name && props.title} : {props.name && props.name}{props.name && ','} </Text>
            </View>
        )
    

export default UserDataItem;
             