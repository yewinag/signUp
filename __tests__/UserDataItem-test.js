import 'react-native';
import React from 'react';
import UserDataItem from '../src/components/UserDataItem';

import renderer from 'react-test-renderer';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

Enzyme.configure({ adapter: new Adapter() });

it('test render Userdataitem correctly', () => {
  const tree = renderer.create(<UserDataItem />).toJSON();
  expect(tree).toMatchSnapshot();
});