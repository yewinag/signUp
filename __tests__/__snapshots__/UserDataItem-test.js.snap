// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test render SignUpForm correctly 1`] = `
<View>
  <Text>
       
     
     : 
     
  </Text>
</View>
`;

exports[`test render Userdataitem correctly 1`] = `
<View>
  <Text>
       
     
     : 
     
  </Text>
</View>
`;
